pods-beaver-themer.php

```diff
@@ -218,38 +222,50 @@ function pods_beaver_loop_settings_before_form( $settings ) {
  *
  * @return object
  *
- * @since 1.0
+ * @since 1.1
  */
 function pods_beaver_loop_before_query_settings( $settings ) {
 
 	if ( empty( $settings->use_pods ) || 'no' === $settings->use_pods ) {
 		return $settings;
 	}
 
+/*	global $wp_query, $wp_the_query, $paged;
+
+	$flpaged = $wp_the_query->get( 'flpaged'. FLBuilderLoop::$loop_counter );
+	$page_qv = $wp_the_query->get( 'page' );
+	$paged_qv = $wp_the_query->get( 'paged' );
+	$loop_counter = FLBuilderLoop::$loop_counter;
+	$paged_beaver = FLBuilderLoop::get_paged();
+	$max_page = $wp_query->max_num_pages;
+*/
+
+
 	$ids = array();
 
 	$find_params  = array();
 	$field_params = array();
 
-	$pod = null;
+	$pod = PodsBeaverPageData::get_pod( $settings );
 
 	if ( 'pods_relation' === $settings->use_pods && ! empty( $settings->pods_source_relation ) ) {
 		$field_params = array(
-			'output' => 'id',
 			'name'   => trim( $settings->pods_source_relation ),
 		);
-
-		$pod = PodsBeaverPageData::get_pod();
 	} elseif ( 'pods_settings_relation' === $settings->use_pods ) {
-		$pod = PodsBeaverPageData::get_pod( $settings );
-	} elseif ( 'pods_advanced' === $settings->use_pods && ! empty( $settings->pods_where ) ) {
+		$field_params = array(
+			'name'   => trim( $settings->field ),
+		);
+	} /*elseif ( 'pods_advanced' === $settings->use_pods && ! empty( $settings->pods_where ) ) {
 		$find_params = array(
 			'where' => trim( $settings->pods_where ),
 			'limit' => - 1,
 		);
+	}*/
+
+	$find_params = apply_filters('pods_beaver_loop_settings_find_params', $find_params, $settings, $pod);
```

Comment:
Hook docs for this? https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters